tools/basictool.py

```diff
@@ -0,0 +1,113 @@
+"""CLI tool for loopchain"""
+import inspect
+import os
+import traceback
+from collections import OrderedDict
+from pathlib import Path
+from sys import platform
+
+from earlgrey import MessageQueueService
```

Comment:
`Earlgrey` is not in requirements.

Reply:
This tool cannot be run independently. This tool run in loopchain's venv with some additional requirements. so Its requirements has only additionals.